handlers/handlers.go

```diff
@@ -67,3 +70,11 @@ func dockerPs(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, ctrs)
 }
+
+func newProject(c echo.Context) error {
+	_, err := models.GetDB()
```

Comment:
fine for now, in the future lets void opening connections on every request. we can keep a persistent connection open.